setup/environment-linux-aarch64.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
   - pandas=1.2.1
   - pip=21.1.2
   - prompt_toolkit=3.0.20
+  - pydantic=1.9.0
```

Comment:
I would suggest specifying the version up the the major number only (1.9). In that way if a new minor release with fixes is published we will start using it right away.

Reply:
Good point, I updated the environment files